### PR TITLE
Add Urdu and various African languages

### PR DESCRIPTION
**Pull request by bslaught**
## Why?

We want to add these:
- Adlam script (Fulani language) - `data/ff`
-  Ethiopic script (Ethiopian language) - `data/am`
- N’Ko script (Mandé languages) - `data/nqo`
- Osmanya script (Somali language) - `data/so-Osma`
- “Pan African (Latin & IPA)” (many languages) - `data/panafrican`
- Tifinagh script (Berber languages) - `data/shi`
- Urdu - `data/ur`
- Vai script (Vai language) - `data/vai`

## What?

All of the data files except `data/panafrican` and `data/so` are the exact output of the Speakeasy Converter on CLDR 43.1.0.

For `data/so`, I started with the output from the Speakeasy Converter and added the characters from the [Osmanya code chart](https://unicode.org/charts/PDF/U10480.pdf).

`data/panafrican` comes from a script I ran on the following file: [Pan African Unicodes.txt](https://github.com/typekit/speakeasy/files/12034672/Pan.African.Unicodes.txt)

I verified I'm able to load these languages into typekit/typekit.

### CODE REVIEW

File: data/so [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- !ruby/range 74..78       # J,K,L,M,N
+- !ruby/range 81..84       # Q,R,S,T
+- !ruby/range 87..89       # W,X,Y
+- !ruby/range 98..100      # b,c,d
+- !ruby/range 102..104     # f,g,h
+- !ruby/range 106..110     # j,k,l,m,n
+- !ruby/range 113..116     # q,r,s,t
+- !ruby/range 119..121     # w,x,y
+- !ruby/range 66688..66717 # 𐒀,𐒁,𐒂,𐒃,𐒄,𐒅,𐒆,𐒇,𐒈,𐒉,𐒊,𐒋,𐒌,𐒍,𐒎,𐒏,𐒐,𐒑,𐒒,𐒓,𐒔,𐒕,𐒖,𐒗,𐒘,𐒙,𐒚,𐒛,𐒜
+- !ruby/range 66720..66729 # 𐒠,𐒡,𐒢,𐒣,𐒤,𐒥,𐒦,𐒧,𐒨,𐒩
```
Comment (cjchapman) [inline data/so:17]: Please remove line 17, those are digits:
```
!ruby/range 66720..66729 # 𐒠,𐒡,𐒢,𐒣,𐒤,𐒥,𐒦,𐒧,𐒨,𐒩
```

Comment (cjchapman) [reply data/so:17]: Also, please change the file name to `so-Osma`, the anglicized & native names to `Somali (Osmanya script)`, and remove the Latin ranges.

**Comment by cjchapman** [comment]
I did a quick first pass at a review this evening. I plan to take another look at it tomorrow when I'm not so tired. 😀

**Review by cjchapman — CHANGES_REQUESTED** [review]
I don't see anything else today, so there's just the changes I requested last night.

File: data/so (hunk 1 above)
Comment (bslaught) [reply data/so]: Changed in https://github.com/typekit/speakeasy/pull/43/commits/11141eb307d0463e1ec44a694faf671288ef0018.

File: data/so-Osma [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+anglicized_name: Somali
+native_name: Soomaali
```
Comment (cjchapman) [inline data/so-Osma:4]: please change this to:
```
anglicized_name: Somali (Osmanya script)
native_name: Soomaali (Farta Cismaanya)
```
(I'm sorry that I didn't give enough information in my previous request about this.)

**Review by cjchapman — CHANGES_REQUESTED** [review]

File: data/so-Osma (hunk 2 above)
Comment (bslaught) [reply data/so-Osma]: Added in https://github.com/typekit/speakeasy/pull/43/commits/1f88ff197b42fd35727e3d77ba68b7adfa20b878.

File: data/so-Osma [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- !ruby/range 66..68       # B,C,D
+- !ruby/range 70..72       # F,G,H
+- !ruby/range 74..78       # J,K,L,M,N
+- !ruby/range 81..84       # Q,R,S,T
+- !ruby/range 87..89       # W,X,Y
+- !ruby/range 98..100      # b,c,d
+- !ruby/range 102..104     # f,g,h
+- !ruby/range 106..110     # j,k,l,m,n
+- !ruby/range 113..116     # q,r,s,t
+- !ruby/range 119..121     # w,x,y
```
Comment (cjchapman) [inline data/so-Osma:15]: Please remove these Latin ranges. (I'm sorry I wasn't clear in my earlier request about this.)

**Review by cjchapman — CHANGES_REQUESTED** [review]

File: data/so-Osma (hunk 3 above)
Comment (bslaught) [reply data/so-Osma]: Removed in https://github.com/typekit/speakeasy/pull/43/commits/f767ec078130c0d47b7a588681896c2fc2fdda2c.

**Review by cjchapman — APPROVED** [review]
👍

**Pull request merged**